frontend/app/views/event/masterclass.scala.html

```diff
@@ -7,7 +7,9 @@
 
 @import model.RichEvent.MasterclassEvent
 
-@main("Masterclasses", htmlClass="events-bg theme--masterclasses", pageInfo=pageInfo) {
+@main("Masterclasses", pageInfo=pageInfo) {
+
+    <main role="main" class="l-constrained">
```

Comment:
Being fussy here, but everything inside this `<main>` element needs to be indented one level.